### PR TITLE
Add warning to grafana link if special addons aren't enabled

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Add warning to grafana link if node-exporter and/or kube-state-metrics addons aren't enabled

### Which issue(s) this PR fixes

Fixes https://github.com/kubermatic/kubermatic/issues/7961

### Special notes for your reviewer

grafana-warning

### Release note

```release-note
NONE
```

### CODE REVIEW

File: src/app/cluster/cluster-details/mla/alertmanager-config/component.ts [hunk 1]
```diff
@@ -34,6 +35,11 @@ export enum Type {
   Grafana = 'Grafana',
 }
 
+export enum AddonType {
+  NodeExporter = 'node-exporter',
+  KubeStateMetrics = 'kube-state-metrics',
+}
```
Comment (maciaszczykm) [inline src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:41]: Is it certain that these two addons will be always available to install?

Comment (kgroschoff) [reply src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:41]: Seems like they have to be installed manually during kkp installation.
MLA also needs some configuration and docs are explaining which addons you need to install https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/monitoring_logging_alerting/user_cluster/admin_guide/#addons-configuration

Do you think we should add some more information about these addons?

Comment (maciaszczykm) [reply src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:41]: I think yes. I'm not sure if everyone will be aware of that.

Comment (kgroschoff) [reply src/app/cluster/cluster-details/mla/alertmanager-config/component.ts:42]: @maciaszczykm added a warning only if addons aren't installable

**Comment by kgroschoff** [comment]
@maciaszczykm rebased, PTAL
/hold cancel

**Pull request merged**